### ISSUE TITLE
New BASE table in Noto Sans Japanese

### ISSUE THREAD

**Issue by chrissimpkins** [google/fonts#5503]
font family: Noto Sans Japanese
version: 2.004-H2
related PR: https://github.com/google/fonts/pull/5469

@punchcutter Hi Zachary!  We noticed that there is a new BASE table in Noto Sans Japanese.  Will you please let us know what the rationale was for this addition?  We're trying to understand where users might find new baseline changes with the update as it is pushed into production.  I ok'd the merge to sandbox so that we can run internal testing on instance sizes and would like to address this issue before we move this version into production.  Thanks!

cc @simoncozens @RosaWagner @m4rc1e @vv-monsalve

**Comment by punchcutter** [comment]
There is no new BASE table. It's the same as it ever was. What are you seeing that looks different?

**Comment by chrissimpkins** [comment]
Thank you very much Zachary.  This information came from automated testing of the changes landing in the new release on the repository vs. the last production release in our web API.  I learned after I posted this that our internal pipeline subsetter strips BASE out which explains why this appears to be a "new" table.  There is no change here.

It remains an open question if we *should* remove it in the files distributed through the web API.
